environment/deployments/science-platform/cloudsql/main.tf

```diff
@@ -1,28 +1,33 @@
-module "private-postgres" {
+# Butler Registry DP02
+module "db_butler_registry_dp02" {
```

Comment:
I might be getting confused but I think this change will remove this registry DB in `science-platform-stable`:
<img width="376" alt="image" src="https://github.com/user-attachments/assets/fbe9ba81-b480-4d3b-81a1-7ce39f559762">

That database in `science-platform-stable` is currently used for the production RSP.  It's only the dev/int ones that are using `panda-dev`.

I was hoping we would set up a new instance parallel to the existing one, so I could set it up and switch over before taking the old one down.

Reply:
You are correct.  Thanks for the catch.  I will update to make the removal conditional.

Reply:
Updated to leave in.  Conditional did not work for an existing resource deployed with a module.

Reply:
Leaving it in for all environments is fine with me -- we don't really need the conditional.  I think what I am going to do is:

1. Set up the new ones
2. Shut down the old ones but don't delete them immediately
3. Delete the old ones after a month or two

So we can just leave them all up for now, then delete them all at the same time, when the time comes.

For future reference I think there is a hack you can do with the `count` parameter and `moved` blocks to retroactively make the module conditional.  It's been a minute since I've terraformed so I might be misremembering but it's something like:
```
module private-postgres {
    count = var.something ? 1 : 0
    ...
}

moved {
  from = private-postgres
  to   = private-postgres[0]
}

```